Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -368,6 +368,52 @@ def sign_binary(infile, stitch_dir, cfg_var):
     hash = bytearray(open('output/input/pubkey'+'.hash','rb').read())
     print(''.join('%02X ' % b for b in hash))
 
+def update_tpm_type(tpm, tree):
```

Comment:
I think we can use python dictionary to optimize the code below to make it easier for maintenance.

Reply:
Will do